pkg/cluster/nodes/node.go

```diff
@@ -218,21 +219,28 @@ func (n *Node) KubeVersion() (version string, err error) {
 }
 
 // IP returns the IP address of the node
-func (n *Node) IP() (ip string, err error) {
+func (n *Node) IP(isIPv6 bool) (ip string, err error) {
+	// Check the IP version
+	cachedIP := &n.nodeCache.ip
+	IPAddress := "IPAddress"
+	if isIPv6 {
+		cachedIP = &n.nodeCache.ipv6
+		IPAddress = "GlobalIPv6Address"
```

Comment:
do we need to add this separate string?

Reply:
yeah, this is the snippet of a `docker inspect`output
```   
                   "IPAddress": "172.17.0.2",
                    "IPPrefixLen": 16,
                    "IPv6Gateway": "2001:db8:1::1",
                    "GlobalIPv6Address": "2001:db8:1::242:ac11:2",
 ```

we need to use a different string depending on the protocol